src/Manopt.jl

```diff
@@ -522,7 +522,12 @@ export SmoothingTechnique, LinearQuadraticHuber, LogarithmicSumOfExponentials
 #
 # Stepsize
 export Stepsize
-export AdaptiveWNGradient, ConstantLength, DecreasingLength, Polyak
+export AdaptiveWNGradient,
```

Comment:
Since we (ok only since yesterday or so) use Runic, we can also do this in two lines.